### ISSUE TITLE
A few more questions

### ISSUE THREAD

**Issue by kilroy98** [lukasz1992/asuswrt-xiaomi#133]
Repository contains files with translations into other languages, but only English and Chinese are available for selection in the web interface. Is this a bug or are all other languages disabled on purpose?

And about `UUAccelerator.asp`, maybe hide it from menu when it is not included in firmware? Or hiding it is causing bugs?

Maybe then add only to nand_ids.c? I found this data in other github repositories, I think that everything is correctly indicated for `nand_ids.c`. Or is this file not used in the firmware?
For my memory, I found the data here (the file name is different, but I think it's the same nand_device_list.h): https://github.com/keenetic/kernel-49/blob/134030ace9cdaadb969bdc4aa0e967d2cd7493f2/drivers/mtd/ralink/mt7621_nand.c#L126

I think the data is correct and at least this can be added.

nand_device_list.h:
```
{0x98F1, 0x8015F2, 4, 8, 128, 128, 2048, 64, 0x00844333, "TC58BVG0S3H", 0},
```

nand_ids.c:
```
{"TC58BVG0S3H 1G 3.3V 8-bit",
		{ .id = {0x98, 0xf1, 0x80, 0x15, 0xf2, 0x16, 0x08, 0x00} },
		  SZ_2K, SZ_128, SZ_128K, 0, 8, 64},
	{"TC58NVG0S3E 1G 3.3V 8-bit",
		{ .id = {0x98, 0xd1, 0x90, 0x15, 0x76, 0x14, 0x01, 0x00} },
		  SZ_2K, SZ_128, SZ_128K, 0, 8, 64},
```
TC58NVG0S3E get from here 
https://github.com/SWRT-dev/rtax53u/blob/a3dd685d2d642885ed2c6db3d752cce4b69d5737/release/linux/mtk/linux-4.4.198/drivers/mtd/nand/nand_ids.c#L32
TC58BVG0S3H ger from here
https://github.com/hardfalcon/VMG8825-D70B/blob/1533d134ea9b8aef615257af0656f47ea068aaf1/platform/iad/kernel/linux-3.10-git/linux-3.10/drivers/mtd/nand/nand_ids.c#L48

**Comment by lukasz1992** [comment]
What exactly do you mean about webif, can you send screenshoot?
Which menu shows you UUAccelerator.asp?

Let's ignore nand parameters - it is only cosmetic during boot, defaults are fine

**Comment by kilroy98** [comment]
Can't take a screenshot from PC at the moment.
![Screenshot_2022-04-28-15-00-19-414_com android chrome](https://user-images.githubusercontent.com/13346898/165747435-28f727a5-b344-4b63-a7a6-079b562b1c46.jpg)

**Comment by lukasz1992** [comment]
What release did the problem start? Can you check if there is any error in Web console in browser?

UUAccelerator is not present and should not be visible

**Comment by kilroy98** [comment]
I discovered this from the first test release, which started to fully load on my router. There are no errors in the browser console. Is that just this, but I think these are Google Chrome errors, not the router's web interface
![bandicam 2022-04-28 16-25-33-752](https://user-images.githubusercontent.com/13346898/165762931-f92cf687-ce5c-4dcc-b9dd-2cdc5d6b4f84.jpg)
![bandicam 2022-04-28 16-25-40-278](https://user-images.githubusercontent.com/13346898/165762939-f64ad6e8-9ffb-44a9-8dcc-d8f06bc0700d.jpg)

**Comment by kilroy98** [comment]
I installed firefox and still the same, only warnings in the browser console :(

**Comment by lukasz1992** [comment]
Does factory reset help?

**Comment by kilroy98** [comment]
No, i did factory reset after flashing latest version of firmware.

**Comment by kilroy98** [comment]
On 4pda, I was advised flash my eeprom backup, which I did. After the reboot, everything fell into place. I do not understand how it is connected and what is broken, but it worked. Before restoring it, I made a backup copy of it, but now I can’t download the full dump, the download of the dump starts and stops at 1.8Mb of 128Mb and the speed drops to 0. I don’t know what’s broken, that I can no longer download the full dump.
